src/wifi_hal_nl80211_utils.c

```diff
@@ -72,6 +72,25 @@ static const wifi_interface_name_idex_map_t static_interface_index_map[] = {
 #endif
 #endif
 
+#ifdef BANANA_PI_PORT // for reference device platforms
+    {0, 0,  "wifi0",     "brlan0",    0,    0,      "private_ssid_2g"},
+    {1, 1,  "wifi1",     "brlan0",    0,    1,      "private_ssid_5g"},
+    {0, 0,  "wifi2",     "brlan1",    0,    2,      "iot_ssid_2g"},
+    {1, 1,  "wifi3",     "brlan1",    0,    3,      "iot_ssid_5g"},
+    {0, 0,  "wifi4",     "brlan2",    0,    4,      "hotspot_open_2g"},
+    {1, 1,  "wifi5",     "brlan3",    0,    5,      "hotspot_open_5g"},
+    {0, 0,  "wifi6",     "br1an4",    0,    6,      "lnf_psk_2g"},
+    {1, 1,  "wifi7",     "brlan3",    0,    7,      "lnf_psk_5g"},
+    {0, 0,  "wifi8",     "brlan4",    0,    8,      "hotspot_secure_2g"},
+    {1, 1,  "wifi9",     "brlan5",    0,    9,      "hotspot_secure_5g"},
+    {0, 0,  "wifi10",    "br1an6",    0,    10,     "lnf_radius_2g"},
+    {1, 1,  "wifi11",    "br1an6",    0,    11,     "lnf_radius_5g"},
+    {0, 0,  "wifi12",    "brlan2",    0,    12,     "mesh_backhaul_2g"},
+    {1, 1,  "wifi13",    "brlan3",    0,    13,     "mesh_backhaul_5g"},
+    {0, 0,  "wifi14",    "brlan2",    0,    14,     "mesh_sta_2g"},
+    {1, 1,  "wifi15",    "brlan2",    0,    15,     "mesh_sta_5g"},
+#endif
```

Comment:
In the above definition the first element is phy index. In case of Banana Pi since there is only one phy, the phy_index should be only one value 0 probably. phy_index of 1 entries are not appropriate.

Reply:
yes.. I agreed.
can you please help me to change the index value .
Also, this code changes will work only for 2G band. For dual band validation, we used JSON file from nvram folder.

Reply:
Removed above lines in latest commit